kolibri/plugins/epub_viewer/assets/src/views/EpubRendererIndex.vue

```diff
@@ -678,6 +707,7 @@
         this.sliderValue = location.start.percentage * 100;
         this.updateCurrentSection(location.start);
         this.currentLocation = location.start.cfi;
+        this.storeVisitedPage(this.currentLocation);
```

Comment:
I feel like we could just call `updateProgress` here too. At the moment, it's not getting updated except on the 30 second timer, whereas if someone is quickly flicking through the pages, that won't be catching it as fast as this would.